projects/WeTek_Hub/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Debug Shell on /dev/ttyS0
+DefaultDependencies=no
+ConditionKernelCommandLine=console=ttyS0,115200
+
+[Service]
+WorkingDirectory=/storage
+Environment="ENV=/etc/profile"
+ExecStartPre=/bin/sh -c 'echo -en "\033[?25h"'
```

Comment:
For me cursor is hidden from time to time (on imx6). This line helps to be visible in little more cases than without. But still doesn't work 100%. Didn't investigate why it happens but it started after PR #44.